app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -47,9 +47,12 @@
     toggleOpenMenu: function () {
       $('.ProvidedByEducatorDropdown').autocomplete("search", "");
     },
-
+    closeMenu: function () {
+      $('.ProvidedByEducatorDropdown').autocomplete('close');
```

Comment:
This is a small improvement, but right now this does a query through the whole DOM with `$` (same for the existing code too).  We can explicitly scope this to the component's DOM node like this: https://facebook.github.io/react/docs/top-level-api.html#reactdom.finddomnode, which makes it impossible to possibly affect other sibling components that match this selector.